test/gtest/cfg.cpp

```diff
@@ -579,3 +581,67 @@ TEST_F(CFGTest, ReachingDefinitionsLoop) {
 
   EXPECT_EQ(expectedResult, getSetses);
 }
+
+TEST_F(CFGTest, StackBitsLatticeFunctioning) {
+  BitsLattice bitsLattice;
+  StackLattice<BitsLattice> stackLattice(bitsLattice);
+
+  StackLattice<BitsLattice>::Element firstStack = stackLattice.getBottom();
+  StackLattice<BitsLattice>::Element secondStack = stackLattice.getBottom();
+
+  for (uint32_t i = 0; i < 4; i++) {
+    BitsLattice::Element temp = bitsLattice.getBottom();
+    temp.setValue(i);
+    firstStack.push(temp);
+  }
+
+  for (uint32_t i = 0; i < 4; i++) {
+    BitsLattice::Element temp = bitsLattice.getBottom();
+    temp.setValue(i);
+    secondStack.push(temp);
+  }
+
+  EXPECT_EQ(stackLattice.compare(firstStack, secondStack),
+            LatticeComparison::EQUAL);
+
+  secondStack.pop();
+  secondStack.pop();
```

Comment:
It will probably be nice later to have `pop` return the element rather than having to do `stackTop` and `pop` separately.